editor/post-title/index.js

```diff
@@ -76,8 +85,19 @@ class PostTitle extends Component {
 		this.setState( { isSelected: false } );
 	}
 
-	handleClickOutside() {
-		this.setState( { isSelected: false } );
+	setFocused( focused ) {
+		this.setState( { hasFocusWithin: focused } );
+		if ( focused ) {
+			this.props.clearSelectedBlock();
```

Comment:
Why is this needed? Ideally the title focus handler shouldn't need to deal with block selection (i.e. it should be block focus leave events clearing its selected state).

Reply:
When I was testing it before, that wasn't working. But that was an earlier version. I'm happy to remove it and try the blur of the other blocks.

Reply:
Removed this.